### PR TITLE
[TOOLS-2031] Multi module modularization

### PR DESCRIPTION
**Pull request by taksan**
PR to run the tests

### CODE REVIEW

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpringContextBridge.java#L35](https://github.com/objective-solutions/taskboard/blob/40e96ae557051adf7495d272af7e84852c9dfd6a/core/src/main/java/objective/taskboard/config/SpringContextBridge.java#L35): Synchronize this lazy initialization of 'applicationContext' [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS2444)
1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/40e96ae557051adf7495d272af7e84852c9dfd6a/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/6f49cd64b15b36508a8fb84e7fbdb97cfe77439c/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/9e7b8bd66712641f296dd230ab58c4ce737f95fa/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/7c1d9a361b7bdd763db5b8ea471ed86b452e42f3/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/26423b6ad456fc9b2720ce763a6cf6fc1301d90e/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by robersongomes** [comment]
Is there any reason you didn't extract the cluster part to a new module?

File: core/src/main/resources/static/elements/taskboard/inline-badge.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!--
```
[the remark this answers is not in the document]
Comment (taksan) [reply core/src/main/resources/static/elements/taskboard/inline-badge.html]: My mistake. I will remove from core

**Comment by taksan** [comment]
As for the cluster, I intend to separate, but we have two parts: the retrofit and the "followup cluster". I think it needs some package moving and some renames, so I left to extract in a second iteration

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MainClassRunner.java#L15](https://github.com/objective-solutions/taskboard/blob/0c77aa220e809e03c323cd171cd79a4386ad0a3b/application/src/main/java/objective/taskboard/MainClassRunner.java#L15): Move the "--CacheGenerate" string literal on the left side of this string comparison. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1132)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**